compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/EncodeV1.hs

```diff
@@ -8,19 +8,20 @@ module DA.Daml.LF.Proto3.EncodeV1
   , encodePackage
   ) where
 
-import           Control.Lens ((^.), (^..), matching)
+import           Control.Lens ((^.), matching)
 import           Control.Lens.Ast (rightSpine)
-import           Control.Monad.Reader.Class
+import           Control.Monad.State
```

Comment:
This should almost certainly be `Control.Monad.State.Strict`.

Reply:
Good catch. Thanks.